grapple/urls.py

```diff
@@ -24,7 +25,9 @@ def graphiql(request):
 
 
 # Traditional URL routing
-urlpatterns = [url(r"^graphql", csrf_exempt(GraphQLView.as_view()))]
+urlpatterns = [
```

Comment:
suggestion: time to switch to `path`

otherwise, LGTM

Reply:
Updated!